zilliqa/src/exec.rs

```diff
@@ -367,6 +368,7 @@ impl State {
         chain_id: u64,
         current_block: BlockHeader,
         inspector: I,
+        skip_base_fee_check: bool,
```

Comment:
I'm not all that happy about this being a boolean, and a negative-sense boolean at that - suggest change to an enum to make it obvious what you're doing so that accidental `true`s here stand out and folks spot them in code review. A failure to do so could be bad .. (not least because it would make code to replay history once you'd fixed it a pretty awful thing to have to write)

Reply:
Done :+1: